packages/react-core/src/components/ChipGroup/ChipGroup.tsx

```diff
@@ -156,7 +160,14 @@ export class ChipGroup extends React.Component<ChipGroupProps, ChipGroupState> {
               ))}
               {numChildren > numChips && (
                 <li className={css(styles.chipGroupListItem)}>
-                  <Chip isOverflowChip onClick={this.toggleCollapse} component="button">
+                  <Chip
+                    isOverflowChip
+                    onClick={event => {
+                      this.toggleCollapse();
+                      onOverflowChipClick(event);
```

Comment:
Shouldn't calling this be  this be conditional on wether it as an overflow chip?

Reply:
From what I understand, line 156 in ChipGroup.tsx is generating all the non-overflow chips passed through as children, and then this last Chip is by definition always the overflow chip This chip is only rendered if numChildren > numChips (i.e., if we have overflow).  Here, there is the `isOverflowChip` which is shorthand for `isOverflowChip={true}`. 

So this code here should be only applicable to the overflow chip.